### DISCUSSION TITLE
Problem with the fan when returning

### DISCUSSION THREAD

**Discussion by wjr88** [T-Troll/alienfx-tools#176] [General]
It's been a while since I noticed that after an update whenever the Windows screen locked, when I came back the fans slowed down and no longer returned to the configuration established by me (100%). To get back to normal just restarting the Alien Fan but the problem is that if, for example, you move the mouse accidentally and the screen comes back, the fans slow down and if I don't see it right away the temperatures rise a lot, which can cause problems long-term. To avoid this, I always kept the app out of date, but I had to update it because it was no longer opening. Does anyone know how to do this in this case? I've tried everything. Thanks.

**Comment by T-Troll** [comment]
Ok, let's check.
First, what version working normally and which one have this issue?
Second, it can be correct - you DO NOT set fans to 100% (use other power profile for this). You just set additional RPM above BIOS default. So if temperature lower, fans will be slowdown (and stop if below passive point).

PS: Looks strange for me - app continue working if screen locked or off. Maybe your system fall to sleep mode?

**Reply by wjr88** [reply]
Thanks for the clarifications!
The version that didn't give a problem is from March or April, I don't know for sure anymore because I installed the other one on top. I use it for mining and the Windows issue it doesn't go into sleep mode. The fans continue to function normally after the screen turns off and Windows is locked. The problem happens when you move the mouse, when you move the fans they simply slow down and don't go back to what was configured. Dell G15.

**Reply by T-Troll** [reply]
Wow!
What OS?
In fact, this means you have issues with Windows in itself.
The main difference between old 6.x.x.x and current 7.x.x.x is first use direct ACPI access (which brings a lot of issues due to Windows 11 "security"), and second one use WMI calls.
So seems like mouse move broke WMI(!).

Also, check no other app (like AWCCService or HWINFO) running, it can interfere.
